cpu/cortexm_common/cortexm_init.c

```diff
@@ -29,7 +29,11 @@
 /**
  * Interrupt vector base address, defined by the linker
  */
+#ifndef SOFTDEVICE_PRESENT
 extern const void *_isr_vectors;
+#else
+#undef _isr_vectors
+#endif
```

Comment:
Maybe this block can belong to the nrf52 cpu code?

Reply:
I'll give it a try